include/stdio.h

```diff
@@ -259,6 +309,8 @@ FAR char *tmpnam(FAR char *s);
 FAR char *tempnam(FAR const char *dir, FAR const char *pfx) malloc_like;
 int       remove(FAR const char *path);
 
+FAR FILE *lib_get_stdio_stream(int fd);
```

Comment:
let's move lib_get_stream from nuttx/lib/lib.h to here:
```suggestion
FAR FILE *lib_get_stream(int fd);
```